app/pages/dashboard/buildGenerateAGPImagesFunction.js

```diff
@@ -0,0 +1,58 @@
+import _ from 'lodash';
+import { utils as vizUtils } from '@tidepool/viz';
+import Plotly from 'plotly.js-basic-dist-min';
+import * as actions from '../../redux/actions';
+
+const buildGenerateAGPImages = (dispatch) => {
```

Comment:
Copied from `app/pages/patientdata/patientdata.js:922`. The fake "props" object is just to prevent needing to change the code.